### PR TITLE
Fix duplicated module headers

### PR DESCRIPTION
**Pull request by ruflin**
Closes https://github.com/elastic/beats/issues/7643

### CODE REVIEW

**Comment by ruflin** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
This fixes the duplication of the **module** doc. But it does not yet fix the duplication of the **metricset** docs. For example I now only see one doc for "Munin module" but two (duplicated) docs under it for the node metricset.

screen shot 2018-07-20 at 6 46 47 am

To fix the duplication in the metricset docs I think you also need to edit `{module}/{metricset}/_meta/docs.asciidoc` and remove the `== {module} {metricset} MetricSet` line from the top.

File: metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the `apiserver` metricset of the Kubernetes module.
```
Comment (ruflin) [inline metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc]: @exekias I wonder if you left this out on purpose?

Comment (exekias) [reply metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc]: probably not, :+1: to having it

**Comment by ycombinator** [comment]
Sorry @ruflin but I just happened to notice while clicking around that the `traefik` module has the same duplication issue with its metricset. Mind fixing that one in this PR too?

**Comment by ruflin** [comment]
@ycombinator Fixed. I also started to wonder why the new modules / metricsets have this issue and realised it's a left over in the generator we forgot to remove when we started to auto generate the titles. I now also cleaned up the generator.

**Comment by ycombinator** [comment]
LGTM.

**Comment by dedemorton** [comment]
Removing the `needs_backport` label because the changes are already in 6.x and 6.4 (I guess they were part of the auto merge.) I've created a separate backport PR for 6.3 (#7650), so I think we're covered.

**Pull request merged**